src/GraphQL.Conventions/Extensions/GraphQLBuilderExtensions.cs

```diff
@@ -0,0 +1,65 @@
+#nullable enable
+
+using System;
+using System.Collections.Generic;
+using GraphQL.DI;
+
+namespace GraphQL.Conventions
```

Comment:
Remember that by convention (😄 ) all extension methods on `IGraphQLBuilder` live in `GraphQL` root namespace.

Reply:
Good catch

- #256